flake.nix

```diff
@@ -35,6 +35,7 @@
           autoconf
           cmake
           clang
+          curl
```

Comment:
I'm trying to make it so that `curl` command is available to the command `nix build '.#kevm-test'`, but this doesn't seem to have done the trick. @Baltoli any quick suggestions?

Reply:
I seem to have made it past teh `curl` issue.

Reply:
But now I get this one: 
![image](https://user-images.githubusercontent.com/12721972/200941592-f754df8d-fb7b-4358-915f-53433a6d52b4.png)

Are nix derivations not allowed to reach the network?